pegasus/test/fixtures/sites/code.org/public/div_brackets.md

```diff
@@ -0,0 +1,16 @@
+---
+# Test YAML header metadata
+x: y
+---
```

Comment:
Given this test, is something like this still a valid YAML header given it's not just strings: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/challenge.md

Reply:
Any valid YAML content that parses to a hash is considered a valid YAML header, so yes, something like that one is still a valid YAML header. (The `test_pegasus_documents` test covers this file, so if it wasn't valid the test would be failing.)